client/src/components/UserAuth.vue

```diff
@@ -36,6 +36,8 @@
 </template>
 
 <script>
+import { mapState, mapActions } from "pinia";
+import { useStore } from "../store.js";

```

Comment:
prej nism opazu, ampak pomoje bi blo boljš imet relativne poti / pathe do fajlov.. recimo `@/store.js`... lahko pogledaš vite alias kako se nastavi, da pol recimo `@` kaže na src folder..

Reply:
No kul, nism vedu da je to sploh možno...počekiram

Reply:
A potem imam napisano samo npr. @/store.js?
samo če mam pa v components folderju vse komponente, moram pa še vedno dat npr. @/components/BreedsList.vue?
Ker ta @ je kot /src

Reply:
drži.. ampak najprej si moreš narest alias, alpa več aliasov, kokr ti paše, da vite ve kaj ta `@` pomen 👍